### PR TITLE
Tiny fixes for the plot tutorial

### PR DESCRIPTION
**Pull request by core-man**
**Description of proposed changes**

See https://en.wikipedia.org/wiki/Hypocenter:
> A hypocenter (or hypocentre) is the point of origin of an earthquake or a subsurface nuclear explosion.

"the locations of the hypocenters of the earthquakes" seems a little repeated because `hypocenter` is actually the location of an earthquake. So it's better to remove "of the hypocenters".

Related to #983 and #996.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: examples/tutorials/plot.py [hunk 1]
```diff
@@ -41,8 +41,8 @@
 
 
 ########################################################################################
-# We'll use :meth:`pygmt.Figure.plot` method to plot circles on the locations of the
-# hypocenters of the earthquakes.
+# We'll use :meth:`pygmt.Figure.plot` method to plot circles on the locations
```
[the remark this answers is not in the document]
Comment (core-man) [reply examples/tutorials/plot.py:44]: I am not sure which of **epicenter** or **epicentral** should be used here.

[the remark this answers is not in the document]
Comment (meghanrjones) [reply examples/tutorials/plot.py:44]: Maybe 'on the earthquake epicenters' would work (to save us from having to pick)

**Comment by seisman** [comment]
Could you also change `argument` to `parameter` in this tutorial, as discussed in #886.

**Comment by core-man** [comment]
> Could you also change `argument` to `parameter` in this tutorial, as discussed in #886.

Done in fb76425.

**Comment by seisman** [comment]
Line 64: Change "the ``color``" to "the ``color`` parameter".

**Comment by core-man** [comment]
> Line 64: Change "the `color`" to "the `color` parameter".

done in 552aa5c.

**Review by seisman — APPROVED** [review]

**Pull request merged**